db/version_set.cc

```diff
@@ -2176,16 +2176,8 @@ void VersionStorageInfo::EstimateCompactionBytesNeeded(
     level_size += f->fd.GetFileSize();
   }
   // Level 0
-  bool level0_compact_triggered = false;
-  if (static_cast<int>(files_[0].size()) >=
-          mutable_cf_options.level0_file_num_compaction_trigger ||
-      level_size >= mutable_cf_options.max_bytes_for_level_base) {
-    level0_compact_triggered = true;
-    estimated_compaction_needed_bytes_ = level_size;
-    bytes_compact_to_next_level = level_size;
-  } else {
-    estimated_compaction_needed_bytes_ = 0;
-  }
+  estimated_compaction_needed_bytes_ = level_size;
```

Comment:
can we do without changes to pending compaction bytes?

Reply:
Yes, I forgot to revert it.